planning/surround_obstacle_checker/README.md

```diff
@@ -2,16 +2,47 @@
 
 ## Purpose
 
-`surround_obstacle_checker` is a module to prevent moving if any obstacles is near stopping ego vehicle.
-This module runs only when ego vehicle is stopping.
+This module subscribes required data (ego-pose, obstacles, etc), and publishes zero velocity limit to keep stopping if any of stop conditions are satisfied.
```

Comment:
@TakaHoribe update module purpose in https://github.com/autowarefoundation/autoware.universe/pull/878/commits/046ef9b17b4e3c2beb0d9a95b9d8b77a47f0ed18.